cmd/dep/testdata/harness_tests/init/glide/direct-trans-no-conflict/README.md

```diff
@@ -0,0 +1 @@
+Take a direct dependency on a transient dependency where the versions are not conflicted. Resolving should pass.
```

Comment:
If this test starts failing after the other changes I have requested are made, let's ignore it test temporarily ( add a .ignore file extension to the testcase.json file).  The scenario may require additional handling and will be addressed in another issue: #1316.

Reply:
This test does not fail since the existing code only removes transient dependencies read from importers. Since there is also a direct dependency here, the test passes.